testsuite/features/step_definitions/common_steps.rb

```diff
@@ -714,31 +714,47 @@
 # Repository steps
 
 # Enable tools repositories (both stable and development)
-When(/^I enable SUSE Manager tools repositories on "([^"]*)"$/) do |host|
+When(/^I enable (SUSE Manager|Uyuni) tools repositories on "([^"]*)"$/) do |base_product, host|
   node = get_target(host)
   _os_version, os_family = get_os_version(node)
-  if os_family =~ /^opensuse/ || os_family =~ /^sles/
+  case os_family
+  when os_family =~ /^opensuse/ || os_family =~ /^sles/
```

Comment:
That won't match because of the presence of `os_family =~`.
The following will do:
```
when /^(opensuse|sles)/
when /^centos/
when /^ubuntu/
```

Reply:
Fixed, thanks.